elasticapm/utils/disttracing.py

```diff
@@ -133,3 +133,29 @@ def __init__(self, **kwargs):
         super(TracingOptions, self).__init__()
         for k, v in kwargs.items():
             setattr(self, k, v)
+
+
+def trace_parent_from_string(traceparent_string, tracestate_string=None, is_legacy=False):
+    """
+    This is a wrapper function so we can add traceparent generation to the
+    public API.
+    """
+    return TraceParent.from_string(traceparent_string, tracestate_string=tracestate_string, is_legacy=is_legacy)
+
+
+def trace_parent_from_headers(
+    headers,
+    header_name=constants.TRACEPARENT_HEADER_NAME,
```

Comment:
I wonder if we need to expose these header names in the public API. They are all standardized in the W3C spec, so I don't see a valid use case to allow users to override them

Reply:
Honestly I was torn on whether to expose `from_headers()` at all, but you're right that I can at least simplify it by removing those overrides.